Server/rainbow-server/Sources/Application/Constant.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  Constant.swift
+//  Application
+//
+//  Created by Sanjeev Ghimire on 5/9/18.
+//
+
+import Foundation
+
+public struct Constant {
+    
+    public static let NOTIFICATION_MESSAGE_KNOCKED_FROM_TOP: String = "You have been knocked from the top"
+    public static let NOTIFICATION_MESSAGE_KNOCKED_FROM_TOP_TEN: String = "You have been knocked from top ten"
```

Comment:
"You have been knocked from the top ten spots of the leaderboard. Play again to reclaim your spot!"